docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -324,22 +380,58 @@ The table below maps the Bitbucket Cloud and Bitbucket Server roles to the corre
 <sup>1</sup>: Codacy can't distinguish the Bitbucket roles Read and Write because of a limitation on the Bitbucket API.  
 <sup>2</sup>: Joining an organization may need an approval depending on your setting for [accepting new people](changing-your-plan-and-billing.md#allowing-new-people-to-join-your-organization).
 
+## The Organization Manager role
```

Comment:
We are not being consistent with the capitalization of **Organization Manager**... Do we really need this capitalization here? 🤔 
For example, in the [provider tables](https://github.com/codacy/docs/pull/1912/files#diff-5136bf29081de05223273bdfc6066b94a2575b69ea18a9e4fb668d9f6600733aR141) we don't capitalize, we only use sentence case for column titles, which is consistent with the other roles.

Reply:
👍 Good catch. Fixed.